### DISCUSSION TITLE
It's possible to indirect just get count?

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#3183] [Q&A] [answered]
I have a collection of categories and posts. I can get the categories with the post count if I use the View Collection. But can I do it by expanding?

ex: ?expand=post(category):count

... :D

**Comment by ganigeorgiev** [comment, marked as answer]
No, this is not supported. You need to send a separate request(s) or create a helper view collection.

**Reply by hungcrush** [reply]
I got it.
